### DISCUSSION TITLE
Cannot use simialbi\yii2\schemaorg\models\Product as Product because the name is already in use

### DISCUSSION THREAD

**Discussion by toantrungtinh** [simialbi/yii2-schema-org#8] [Q&A] [answered]
Hey Guy! 
I am having the same problem as above.
Can you give me suggestions?

**Comment by simialbi** [comment, marked as answer]
What's the exact problem? Can't you use 2 classes with the name product in the same e.g. Controller? You could import `simialbi\yii2\schemaorg\models\Product` with an alias (see https://www.php.net/manual/en/language.namespaces.importing.php).
E.g.
```php
use simialbi\yii2\schemaorg\models\Product as SchemaProduct;
```
